src/learner.cc

```diff
@@ -888,8 +888,6 @@ class LearnerImpl : public LearnerIO {
       CHECK(tparam_.dsplit != DataSplitMode::kAuto)
         << "Precondition violated; dsplit cannot be 'auto' in distributed mode";
       if (tparam_.dsplit == DataSplitMode::kCol) {
```

Comment:
Can we deprecate `tparam_.dsplit`?

Reply:
For now I'm keeping it here to include possible data splitting scheme in the future.  Let's think about removing it after benchmark for distributed algorithm is done.